Server/src/Server.Infrastructure/Persistence/QueryHandlers/GetBooksInLibraryHandler.cs

```diff
@@ -25,15 +25,31 @@ public GetBooksInLibraryHandler(IMapper mapper, BookBookDbContext dbContext)
     public async Task<PaginatedResponseViewModel<BookInLibraryViewModel>> Handle(
         GetBooksInLibraryQuery request, CancellationToken cancellationToken)
     {
-        var query = _dbContext.LibraryBooks.AsNoTracking();
+        var query = _dbContext.LibraryBooks
+            .Include(x => x.Book)
+            .ThenInclude(x => x.Authors)
+            .AsNoTracking()
+            .AsSingleQuery();
```

Comment:
Z ciekawości bo chyba pierwszy raz widzę AsSingleQuery, bez podania tego nie leci to jako 1 query nawet jak jawnie piszemy inclue?

Reply:
Szczerze nie do końca wiem, wywalało jakis exception i w logach to było zasugerowane xd